.github/workflows/entropy-check.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: entropy-check
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  entropy-check:
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install python env
+        uses: ./.github/actions/install-python-env
+        with:
+          fetch-depth: 0
+
+      - name: Process repository entropy
+        run: poetry run almanack process_repo_entropy . > entropy_report.txt
+
+      - name: Format Entropy Report
+        id: format_report
+        run: |
+          {
+            echo "REPORT_CONTENT<<EOF";
+            echo "\`\`\`";
+            cat entropy_report.txt;
+            echo "\`\`\`";
+            echo "EOF";
+          } >> "$GITHUB_ENV"
+
+      - name: PR comment
+        uses: thollander/actions-comment-pull-request@v2.5.0
+        with:
+          message: |
+            ${{ env.REPORT_CONTENT }}
+          comment_tag: execution
+          mode: recreate
```

Comment:
FWIW, I think this is failing because the repo currently doesn't allow actions to create pull requests. I'm going to enable the setting, and then perhaps you can retry and see if it succeeds.

Image of the setting below:

![Screenshot 2024-08-13 at 11 39 05 AM](https://github.com/user-attachments/assets/7ee36bbf-78c2-4ff2-8add-e4e81b11c87f)

Reply:
We were running into this issue earlier as well. I have tested this action on my own fork, so it should work once merged. However, Dave figured out that since this repo is within a organization that it prevents us from testing a action "this is a security feature of GitHub to ensure untrusted entities cannot march in and run arbitrary code against a repo and its assets"